api/src/main/java/org/openmrs/module/registrationcore/api/impl/RegistrationCoreServiceImpl.java

```diff
@@ -285,4 +275,17 @@ public boolean supportsPropertyName(String gpName) {
 	public List<String> findSimilarFamilyNames(String searchPhrase) {
 		return getPatientNameSearch().findSimilarFamilyNames(searchPhrase);
 	}
-}
+
+	@Override
+	public String importMpiPatient(String personId) {
```

Comment:
@djazayeri Nope, that is just Integer id.
The reason why I named it "PersonId" instead of "PatientId" is that in OpenEMPI they use person instead of patient name and we receive from OpenEMPI personId which is unique, not patientId.
 So, it is more clear use "person" instead of "patient" .